lms/djangoapps/teams/views.py

```diff
@@ -290,6 +298,7 @@ def get(self, request):
         result_filter = {
             'is_active': True
         }
+        order_by_input = None
```

Comment:
Javascript habit of defining empty variables at the top of a file.  I can move it down to where it's used if that's easier to read.